spec/features/system_admin/lead_partners/add_user_to_lead_partner_spec.rb

```diff
@@ -41,10 +41,18 @@
       when_i_click_the_lead_partner_link
       then_i_see_the_lead_partner_detail_page
       and_i_see_the_user_has_been_added_to_the_lead_partner
+
+      when_i_visit_the_user_page
+      and_i_click_the_remove_link
+      then_i_see_the_remove_lead_partner_page
+
+      when_i_confirm_the_removal
```

Comment:
Not sure if you are aware, there is a spec file for removals [removing_user_access_spec.rb#L20](https://github.com/DFE-Digital/register-trainee-teachers/blob/7168-add-lead-partner-users-in-support-console/spec/features/system_admin/users/removing_user_access_spec.rb#L20)